src/main/java/com/griddynamics/cloud/learning/web/UserController.java

```diff
@@ -0,0 +1,20 @@
+package com.griddynamics.cloud.learning.web;
+
+import com.griddynamics.cloud.learning.service.UserService;
+import com.griddynamics.cloud.learning.web.dto.UserDto;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class UserController {
+
+    private UserService service;
+
+    public UserController(UserService service) { this.service = service; }
+
+    @PostMapping(value = "/sign-up")
+    public ResponseEntity<?> saveUser(UserDto user) {
```

Comment:
-> public UserDto saveUser(UserDto user) {

Reply:
Why do we want to return a UserDto(which represents user-provided values)?

Reply:
Because we want to be awared about user ids and user's roles

Reply:
I've got the idea of user roles. So do I need to use yet another UserDto that will be used to passing information back to the user? (current UserDto is used to receive information from the user)Where are those permissions will come from?

Reply:
It could be same dto